containers/base/javascript/browser/css/dashboard.css

```diff
@@ -125,6 +130,7 @@ a{
 }
 
 .fbd-navigation .mdl-navigation__link {
+  background: white;
```

Comment:
Is that different from the white in the CSS variable?

Reply:
Yeah, the usual white is a bit off-white. #F3F4F5